src/baseline_automatic_pipeline_system/manual_evaluation.py

```diff
@@ -0,0 +1,133 @@
+"""
+This script is for manual assessment of the automatic pipeline. 
+it contains valuable functions to visualize, explore and assess our okr_info output from parse_okr_info. 
+available to import and use in interactive mode.
+"""
+import os, sys
+for pack in os.listdir("src"):
+    sys.path.append(os.path.join("src", pack))
+
```

Comment:
Can you please add function comments throughout this file?

Reply:
Also, probably better to keep a naming convention - what's the difference between ```get```, ```present``` and ```view``` in function names?

Reply:
done